src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -218,7 +218,10 @@ def __call__(self, poller):
                 correlation_message = 'Correlation ID: {}'.format(correlation_id)
             except:  # pylint: disable=bare-except
                 pass
-            self._template_progress(correlation_id)
+            try:
+                self._template_progress(correlation_id)
+            except Exception as ex:  # pylint: disable=broad-except
+                logger.info('%s during progress reporting: %s', getattr(type(ex), '__name__', type(ex)), ex)
```

Comment:
Right, that's why if it can't get the exception's class name, it just returns the exception.  So now you would see:

"Key error during progress reporting" instead of "<'class' Key Error> during progress reporting".